tensorflow/core/kernels/conv_grad_input_ops_3d.cc

```diff
@@ -1151,471 +1151,6 @@ class Conv3DBackpropInputOp<GPUDevice, T> : public OpKernel {
   bool cudnn_use_autotune_;
 };
 
-<<<<<<< HEAD:tensorflow/core/kernels/conv_grad_ops_3d.cc
```

Comment:
It looks like this file split from conv_grad_filter_ops.cc?
Did you verify any differences we had there? There must have been a conflict hence the need for user input. Can you verify we're not missing something in the split?